linuxgsm.sh

```diff
@@ -15,9 +15,10 @@
 
 # Debugging
 if [ -f ".dev-debug" ]; then
-	exec 5>dev-debug.log
```

Comment:
This is by design. dev-debug is overwritten for each command. Change not required

Reply:
This make it very nice to get a complete log of command sequence, would you be fine with something like this:
```bash
if [ -f ".dev-debug" ]; then
	if grep -qFe 'append' '.dev-debug'; then
		exec 5>>dev-debug.log
		echo "invocation linuxgsm.sh $@ $(date "+%Y-%m-%d-%H:%M:%S:%N")"
	else
		exec 5>dev-debug.log
	fi
	BASH_XTRACEFD="5"
	set -x
fi
```

The very early invocation echo make concurrent commands visible in log.

Reply:
Removed from this pr, as its not necessary just nice to have. Actually this has also some issues, so maybe I can do it cleaner on docker side.